pkg/webui/components/input/index.js

```diff
@@ -180,9 +194,16 @@ class Input extends React.Component {
       [style.textarea]: type === 'textarea',
     })
 
+    const byteProps =
+      type === 'byte'
+        ? {
+            showPerChar,
+          }
+        : null
```

Comment:
```suggestion
    const passedProps = { 
      ...rest,
      ...(type === 'byte' ? { showPerChar } : {})
    }
```

And then pass that to the component.